src/admin/src/components/page/advisoryDashboard/AdvisoryDashboard.js

```diff
@@ -67,99 +71,113 @@ export default function AdvisoryDashboard({
     };
   }, [isMounted]);
 
-  const fetchPublicAdvisory = async ({ queryKey }) => {
-    const [, selectedParkId] = queryKey;
+  useEffect(() => {
+
+    const fetchParkNames = async () => {
+      setIsLoading(true);
+      const data = await getProtectedAreas(cmsData, setCmsData);
+      const parkNames = data.map((p) => ({
+        label: p.protectedAreaName,
+        value: p.id,
+      }));
+      setParkNamesData(parkNames);
+      setIsLoading(false);
+    };
+
+    fetchParkNames();
+  }, []);
+
+  useEffect(() => {
+    fetchPublicAdvisory(selectedParkId);
+  }, [selectedParkId]);
+
+  const getCurrentPublishedAdvisories = async (cmsData, setCmsData) => {
+    const advisoryStatuses = await getAdvisoryStatuses(cmsData, setCmsData);
+    if (advisoryStatuses) {
+      const publishedStatus = advisoryStatuses.filter(
+        (as) => as.code === "PUB"
+      );
+      if (publishedStatus && publishedStatus[0]) {
+        await cmsAxios
+          .get(
+            `/public-advisories?_advisoryStatus=${publishedStatus[0].id}&_limit=-1`
+          )
+          .then((res) => {
+            const result = res.data;
+            let publishedAdvisories = [];
+            result.forEach((ad) => {
+              publishedAdvisories = [
+                ...publishedAdvisories,
+                ad.advisoryNumber,
+              ];
+            });
+            if (isMounted.current) {
+              setPublishedAdvisories([...publishedAdvisories]);
+            }
+          })
+          .catch(() => {
+            setHasErrors(true);
+          });
```

Comment:
You're mixing both new ES6 await/async syntax and the old way of promise chaining `.then()`.  I would change it to the new format entirely.

Reply:
done!